index.js

```diff
@@ -3,134 +3,168 @@ const countries = require('./countries');
 module.exports = {
   extend: 'apostrophe-forms-base-field-widgets',
   label: 'Google Address',
-  addFields: [
-    {
-      name: 'googleApiKey',
```

Comment:
This is removing a key field after 1.0.0 is published. I'm sure not many people are using it yet, but that's still a BC break. For what it's worth, this makes a good case for us to use beta versions more to work out issue like this.

Reply:
Yep. However, we can just ship the correction as 2.0.0, and I think that we should do so without shedding a single tear in this case since it is so recent.